### DISCUSSION TITLE
Reolink camera can't communicate (UDP?)

### DISCUSSION THREAD

**Discussion by kamenp** [Ysurac/openmptcprouter#2779] [General]
I installed an OpenMPTCPRouter on a Linksys WRT3200ACM ($50 on Craigslist). It works great, except my Reolink camera is not reachable from the internet.  Their troubleshooting guide has the following paragraph:

> When remotely accessing your camera by using UID, there are no designed ports for it, as the camera uses random UDP ports. So you don't need to forward ports on the router, but the router needs to allow connection via any UDP ports.

So, I suspect that somehow UDP traffic is being blocked somewhere.

Interestingly enough, this worked with a previous OpenMPTCPRouter on a Raspberry Pi. 

Any ideas on what could be the problem?

P.S. The camera is able to send motion alerts, which is probably using a different protocol. Also, I can connect to the camera on the LAN. So, it is connected, just can't punch out to the Internet.

**Comment by Ysurac** [comment]
All traffic to the VPS is blocked by default. If I refer to https://support.reolink.com/hc/en-us/articles/360003544134 you only need to forward ports 80 and 1935.

**Reply by kamenp** [reply]
That is a different method. I couldn't get that one to work either, but that is a separate story.

The "UID-based" connection is also known as "P2P" or "peer-to-peer". This method has known security vulnerabilities, but I don't really care for my use case. The camera monitors access to a publicly visible location. I don't care if a hacker can see it.

The P2P protocol is based on UDP and UDP seems to be blocked somewhere.

Another symptom is that NTP doesn't work (NTP is also based on UDP).
